index.html

```diff
@@ -70,4 +70,168 @@ <h3 class="supporting-text">I’am a Software Developer</h3>
+                                <li><a href="@" class="btn btn-car">Ruby</a></li>
+                            </ul>
+                    </div>
+                            <button class="card-btn">See Project</button>
+                </div>
+
+                <div class="card item4">
+                    <div class="text-block">
+                        <h3 class="new">Profesional Art<br>Printing Data</h3>
+                        <p class="card-description">A daily selection of privately 
+                            personalized reads; no accounts or 
+                            sign-ups required. has been the industry's standard</p>
+                            <ul class="lang">
+                                <li><a href="@" class="btn btn-car">html</a></li>
+                                <li><a href="@" class="btn btn-car">bootstrap</a></li>
+                                <li><a href="@" class="btn btn-car">Ruby</a></li>
+                            </ul>
+                    </div>
+                            <button class="card-btn">See Project</button>
+                </div>
+
+                <div class="card item5">
+                    <div class="text-block">
+                        <h3 class="new">Profesional Art<br>Printing Data</h3>
+                        <p class="card-description">A daily selection of privately 
+                            personalized reads; no accounts or 
+                            sign-ups required. has been the industry's standard</p>
+                            <ul class="lang">
+                                <li><a href="@" class="btn btn-car">html</a></li>
+                                <li><a href="@" class="btn btn-car">bootstrap</a></li>
+                                <li><a href="@" class="btn btn-car">Ruby</a></li>
+                            </ul>
+                    </div>
+                            <button class="card-btn">See Project</button>
+                </div>
+
+                <div class="card item6">
+                    <div class="text-block">
+                        <h3 class="new">Profesional Art<br>Printing Data</h3>
+                        <p class="card-description">A daily selection of privately 
+                            personalized reads; no accounts or 
+                            sign-ups required. has been the industry's standard</p>
+                            <ul class="lang">
+                                <li><a href="@" class="btn btn-car">html</a></li>
+                                <li><a href="@" class="btn btn-car">bootstrap</a></li>
+                                <li><a href="@" class="btn btn-car">Ruby</a></li>
+                            </ul>
+                    </div>
+                            <button class="card-btn">See Project</button>
+                </div>
+            </div>
+            </div>
+    </section>
+
+        <section class="about-section">
```

Comment:
- [ ] You made mention that you need help 👏  adding a background image, but when I check your CSS file in the headline section you did in the last milestone, you added a background image using `background: hsl(20, 6%, 10%) url(images/bgvector.svg) no-repeat;`. In the `about-section`, line `321` of your CSS, you have used `background: url(./images/Rectanglebg.png);` for background image but it does not seem to work. I will suggest in debugging this, start by applying same background image you used in the `headline` section sees if it works. From there you will definitely figure out what is wrong or why the background image you're using is not working. ✊ 
Taking a close look at the `paths` for the background images you have used in the `headline` section and `about-section`, one can notice a difference meanwhile both images are found in the `images` folder. `(./images/.....) vs (images/.....)` Please kindly check your path to make sure it is rendering.